src/components/AppShell.jsx

```diff
@@ -0,0 +1,129 @@
+import {
+  AppShell,
+  Burger,
+  Footer,
+  Header,
+  MediaQuery,
+  Menu,
+  Navbar,
+  Stack,
+  Text,
+  UnstyledButton,
+  useMantineTheme,
+} from "@mantine/core";
+import {
+  IconLogout,
+  IconMoon,
+  IconSettings,
+  IconSunFilled,
+  IconSwitchHorizontal,
+  IconUserCircle,
+} from "@tabler/icons-react";
+import { useState } from "react";
+import { DemoComponent } from "./DemoComponent/DemoComponent";
+
+export default function AppShellDemo() {
+  const theme = useMantineTheme();
+  const [opened, setOpened] = useState(false);
+  const [userMenuOpened, setUserMenuOpened] = useState(false);
+
+  return (
+    <AppShell
+      styles={{
+        main: {
+          background:
+            theme.colorScheme === "dark"
+              ? theme.colors.dark[8]
+              : theme.colors.gray[0],
```

Comment:
Do we need it now?

Reply:
IDK, it was in the example 🙂
Here we could change colors when we decide which colors we should use.